ckanext/canada/tables/service.yaml

```diff
@@ -74,866 +98,1156 @@ resources:
+        Ce champ ne peut excéder une longueur maximale de 250 caractères.
     occurrence: Single
     datastore_type: text
+    max_chars: 250
+    excel_error_formula: 'OR({default_formula},LEN(TRIM({cell}))>250)'
+    form_attrs:
+      style: "width: 100%; display: block"
+      maxlength: 250
 
-  # 3.5
-  - datastore_id: external_internal
+  # DEP 1-6
+  - datastore_id: service_description_en
     label:
-      en: External Service or Internal Enterprise Service
-      fr: Service externe ou service intégré interne
+      en: Service Description (English)
+      fr: Description du service (en anglais)
     description:
-      en: Identifies the service as either an external service or internal enterprise service, as defined in the Policy on Service.
-      fr: Identifie le service en tant que service externe ou service interne intégré au sens des définitions données dans la Politique sur les services.
+      en: Provides a brief description of the service, in plain language.
+      fr: Offre une brève description du service, en langage simple.
     obligation: Mandatory
     excel_required: true
-    format_type: Controlled List
-    validation: This field must not be empty
+    excel_column_width: 80
+    form_required: true
+    format_type: Free text
+    validation:
+      en: |
+        This field must not be empty.
+        This field has a maximum length of 1800 characters.
+      fr: |
+        Cette zone ne doit pas être vide.
+        Ce champ ne peut excéder une longueur maximale de 1800 caractères.
     occurrence: Single
     datastore_type: text
-    excel_full_text_choices: true
-    choices:
-      EI:
-        en: External/Internal Enterprise Service
-        fr: Services externes/internes intégrés
-      EX:
-        en: External Service
-        fr: Services externes
-      IN:
-        en: Internal Enterprise Service
-        fr: Services internes intégrés
+    max_chars: 1800
+    excel_error_formula: 'OR({default_formula},LEN(TRIM({cell}))>1800)'
+    form_snippet: scheming/form_snippets/textarea.html
+    form_attrs:
+      maxlength: 1800
+
+  # DEP 1-7
+  - datastore_id: service_description_fr
+    label:
+      en: Service Description (French)
+      fr: Description du service (en français)
+    description:
+      en: Provides a brief description of the service, in plain language.
+      fr: Offre une brève description du service, en langage simple.
+    obligation: Mandatory
+    excel_required: true
+    excel_column_width: 80
+    form_required: true
+    format_type: Free text
+    validation:
+      en: |
+        This field must not be empty.
+        This field has a maximum length of 1800 characters.
+      fr: |
+        Cette zone ne doit pas être vide.
+        Ce champ ne peut excéder une longueur maximale de 1800 caractères.
+    occurrence: Single
+    datastore_type: text
+    max_chars: 1800
+    excel_error_formula: 'OR({default_formula},LEN(TRIM({cell}))>1800)'
```

Comment:
this part is repeated enough that it might be better as a new `max_chars` feature in recombinant's excel error formula generation

Reply:
got this in recombinant, just need to update all places in YAML. And then test